awesome_owl/static/src/todo_list/todo_item.js

```diff
@@ -0,0 +1,25 @@
+import { Component } from "@odoo/owl";
+
+export class TodoItem extends Component {
+    static template = "awesome_owl.TodoItem";
+    static props = {
+        todo: {
+            type: Object,
+            shape: {
+                id: Number,
+                description: String,
+                isCompleted: Boolean,
+            }
+        },
+        toggleState: Function,
+        removeTodo: Function,
+    };
+
+    onChange() {
+        this.props.toggleState(this.props.todo.id);
```

Comment:
Maybe too much, but I like this way:

```suggestion
        const { toggleState, todo } = this.props;
        toggleState(todo.id);
```

Reply:
I will definitely remember